source/Calamari.AzureServiceFabric/Scripts/AzureServiceFabricContext.ps1

```diff
@@ -15,11 +15,7 @@
 ##   OctopusFabricCertificateFindValueOverride               // The type of FindValue for searching the certificate in the Azure certificate store (use this if you specify a FindType different to 'FindByThumbprint' and do NOT wish to use the client certificate thumbprint value)
 ##   OctopusFabricCertificateStoreLocation                   // The certificate store location (should be 'LocalMachine' by default)
 ##   OctopusFabricCertificateStoreName                       // The certificate store name (should be 'MY' by default)
-##   OctopusFabricAadCredentialType                          // The credential type for authentication
-##   OctopusFabricAadClientCredentialSecret                  // The client application secret for authentication
-##   OctopusFabricAadUserCredentialUsername                  // The username for authentication
-##   OctopusFabricAadUserCredentialPassword                  // The password for authentication
-##   OctopusFabricActiveDirectoryLibraryPath                 // The path to Microsoft.Identity.Client.dll
```

Comment:
These have been replaced with the token below, and used in the C# token retrieval.